doc/introduction/index.rst

```diff
@@ -60,8 +60,9 @@ Zephyr offers a large and ever growing number of features including:
    * *Inter-thread Data Passing Services* for basic message queues, enhanced
      message queues, and byte streams.
 
-   * *Power Management Services* such as tickless idle and an advanced idling
-     infrastructure.
+   * *Power Management Services* such as overarching, application or policy
+     defined, System Power Management, and fine-grained driver defined
+     Device Power Management.
```

Comment:
Sorry to be that guy asking for comma-related changes 🤷‍♂️

```suggestion
   * *Power Management Services* such as overarching, application or
     policy-defined, System Power Management and fine-grained, driver-defined,
     Device Power Management.
```

Reply:
Thanks, I am not a native English speaker, I use commas in a "take it or leave it" manner :)